.build/build-owasp.xml

```diff
@@ -16,30 +16,50 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<project basedir="." name="apache-cassandra-owasp-tasks">
-    <property name="dependency-check.version" value="8.3.1"/>
-    <property name="dependency-check.home" value="${build.dir}/dependency-check-ant-${dependency-check.version}"/>
+<project basedir="." name="apache-cassandra-owasp-tasks"
+         xmlns:unless="ant:unless"
+         xmlns:if="ant:if">
+    <property name="dependency-check.version" value="9.0.2"/>
+    <property name="dependency-check.home" value="${tmp.dir}/dependency-check-ant-${dependency-check.version}"/>
+    <property name="dependency-check.archive.dir" value="${local.repository}/org/owasp/dependency-check-ant/${dependency-check.version}"/>
```

Comment:
Should we use `${local.repository}/org/apache/cassandra/deps/dependency-check-ant` this is a widely used location of other Cassandra dependencies, so I wonder if we should probably have something similar.

Reply:
It is also widely criticized by me😄 Why don't put dependency-check-ant into the right location of the local repository? Look into Maven central repo: https://search.maven.org/artifact/org.owasp/dependency-check-ant - org/owasp/dependency-check-ant seems the right location for this artifact, isn't it?

Reply:
Thanks for the maven link, I should have checked it by myself. +1 then